libsolidity/formal/SMTLib2Interface.cpp

```diff
@@ -64,17 +64,20 @@ void SMTLib2Interface::pop()
 	m_accumulatedOutput.pop_back();
 }
 
-void SMTLib2Interface::declareFunction(string _name, Sort _domain, Sort _codomain)
+void SMTLib2Interface::declareFunction(string _name, vector<Sort> const& _domain, Sort _codomain)
 {
 	// TODO Use domain and codomain as key as well
+	string domain("");
+	for (auto const& _sort: _domain)
+		domain += toSSort(_sort) + ' ';
```

Comment:
Naming the loop variable ``_sort`` made me look for it in the function header...

Reply:
True, I'll change.